doc/ADMIN.md

```diff
@@ -206,3 +222,12 @@ systemctl start synapse.service
 Due of the backup core only feature the data directory in `/home/yunohost.app/synapse` **is not removed**.
 
 Use the `--purge` flag with the command, or remove it manually to purge app user data.
+
+## Slow server
+
+If your server is slow, you can do the following, according to [the official doc](https://matrix-org.github.io/synapse/latest/usage/administration/admin_faq.html#help-synapse-is-slow-and-eats-all-my-ramcpu):
+
+- increase the `SYNAPSE_CACHE_FACTOR` value in your `/etc/default/matrix-__APP__`, `2` is a good value
+  - note that the counterpart is more RAM usage
```

Comment:
well this file is not managed by synapse. We should create a template file which will be in `/etc/matrix-$app/synapse_env` (and declare the usage in systemd unit). And manage synapse cache factor from config panel.